tinkerpop/README.md

```diff
@@ -0,0 +1,97 @@
+Copyright 2016-2023 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+# GafferPop

```

Comment:
Why is the module named `tinkerpop` and not `gafferpop`?

Reply:
I think `uk.gov.gchq.gaffer.tinkerpop` might actually make more sense, and follows what [JanusGraph](https://github.com/JanusGraph/janusgraph/blob/e7b0eea3f37ea60f414d80f3fe1528eb7bf25cfc/janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/JanusGraphBlueprintsGraph.java#L15) do. Having said that, maybe we just call it GafferPop? Not sure